modules/rosa-sts-account-roles-terraform.adoc

```diff
@@ -0,0 +1,217 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-understanding-terraform.adoc
+ifeval::["{context}" == "rosa-understanding-terraform"]
+:tf-full:
+endif::[]
+:_content-type: PROCEDURE
+
+[id="sd-terraform-account-roles_{context}"]
+ifdef::tf-full[]
+= Account roles Terraform example
+endif::tf-full[]
+ifndef::tf-full[]
+= Creating your account-wide IAM roles with Terraform
+endif::tf-full[]
+
+The following example shows how Terraform can be used to create your Amazon Web Services (AWS) Identity and Access Management (IAM) account roles for ROSA.
+
+.Procedure
+
+. Check your AWS account for existing roles and policies by running the following command:
++
+[source,terminal]
+----
+$ rosa list account-roles
+----
+
+. In your terminal, run the following command to export link:https://console.redhat.com/openshift/token[your OCM token]. This value must include the full OCM token:
++
+[source,terminal]
+----
+export RHCS_TOKEN="<your_ocm_token>"
+----
+
+. Optional: run the following command to specify your own account-role prefix that prepends the roles you create by running the following command:
```

Comment:
I think in these cases the first word after "Optional:" should still be capitalized:

```suggestion
. Optional: Run the following command to specify your own account-role prefix that prepends the roles you create by running the following command:
```